### PR TITLE
Added option to specify tooltip directional hint for command bar

### PR DESCRIPTION
**Pull request by adriantran**
Also fixed yarn.lock.

### CODE REVIEW

File: packages/roosterjs-react-command-bar/lib/utils/OutOfBoxCommandBarButtons.tsx [hunk 1]
```diff
@@ -56,6 +55,8 @@ export const RoosterCommmandBarButtonKeys = {
     Code: "code"
 };
 
+const RenderOptionsNoCustomCacheKey: string = null; // buttons that is not menu based doesn't need custom cache keys
```
Comment (pranavgk) [inline packages/roosterjs-react-command-bar/lib/utils/OutOfBoxCommandBarButtons.tsx]: are*

File: packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx [hunk 2]
```diff
@@ -208,14 +212,15 @@ export default class RoosterCommandBar extends React.PureComponent<RoosterComman
...
+        const { strings, calloutClassName, calloutOnDismiss, tooltipDirectionHint } = this.props;
         const className = commandBarButton.className || "";
         const rootClassName = className.split(" ").indexOf(RoosterCommandBarButtonRootClassName) < 0 ? RoosterCommandBarButtonRootClassName : undefined;
         // make a copy of the OOB button template since we're changing its properties
         const button = { ...commandBarButton, className: css(rootClassName, { "first-level": firstLevel }, className) };
 
-        if (!button.onRender && button.onRenderParams) {
-            button.onRender = getIconOnRenderDelegateWithCustomCacheKey(button.key + this._id, ...button.onRenderParams);
+        if (!button.onRender && button.onRenderOptions) {
+            const { customCacheKey = `${button.key}${this._id}` } = button.onRenderOptions;
```
Comment (bochong) [inline packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx]: I'm a little confused on what is happening in this line. What does customCacheKey and button.onRenderOptions become?

Comment (adriantran) [reply packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx]: If customCacheKey (a member of button.onRenderOptions) isn't specified (i.e. it's undefined), we specify a default one. Does that answer your question?

Comment (bochong) [reply packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx]: oh okay i see. yes that makes sense. thanks!

**Pull request merged**